clang/test/CheckedC/inferred-bounds/widened-bounds.c

```diff
@@ -490,7 +489,7 @@ void f20() {
 // CHECK:  [B9]
 // CHECK: upper_bound(q) = 1
 
-  // Declared bounds (INT_MIN) and deref offset (INT_MAX - 1). No sequential deref tests. No widening.
+  // Declared bounds
```

Comment:
I don't understand why you deleted the rest of the comment.  Same goes for other changes in this file.

Reply:
Thanks for catching this. The comments should not have been deleted. Not sure how that happened. I will fix this.